doc/services/pm/device.rst

```diff
@@ -56,10 +54,20 @@ suspended.
    As functions in this context cannot block, transitions that intend to use blocking
    APIs **must** check whether they can do so with :c:func:`k_can_yield`.
 
-This type of device power management can be useful when the application is not
-power aware and does not implement runtime device power management. Though,
-:ref:`Device Runtime Power Management <pm-device-runtime-pm>` is the **preferred**
-option for device power management.
+This type of device power management can be useful in certain scenarios:
+
+ - Simple devices that do not require any blocking operation when suspending and
+   resuming.  This implementation is reasonably simpler than device runtime
+   power management.
+ - For devices that can not make any power management decision and have to be
+   always active. For example a firmware using Zephyr that is controlled by an
+   external entity (e.g Host CPU). In this scenario, some devices have to be
+   always active and should be suspended together with the SoC when requested by
+   this external entity.
+
+It is important to emphasize that his method has, already mentioned,
+drawback and :ref:`Device Runtime Power Management
+<pm-device-runtime-pm>` is the **preferred** option for device power
```

Comment:
This sentence needs to be rewritten (unfinished, s/his/this/, and other grammar/syntax issues)